be/src/column/binary_column.h

```diff
@@ -323,7 +323,7 @@ class BinaryColumnBase final : public ColumnFactory<Column, BinaryColumnBase<T>>
         }
 
         ss << "[";
-        for (size_t i = 0; i < size - 1; i++) {
```

Comment:
why change this code?

Reply:
size-1 overflows to a very large number.

Reply:
it seems size == 0 has been processed
```
        if (size == 0) {
            return "[]";
        }
```

Reply:
I will delete that as it is specially processed.